docs/using/python.md

```diff
@@ -10,164 +10,161 @@ has_children: false
 # Calling the lakeFS API from Python
 {: .no_toc }
 
-The [lakeFS API](../reference/api.md){: target="_blank" } is OpenAPI 2.0 compliant, allowing the dynamic generation of clients from multiple languages.
+The [lakeFS API](../reference/api.md){: target="_blank" } is OpenAPI 3.0 compliant, allowing the generation of clients from multiple languages.
 
-For Python, this example uses [Bravado](https://github.com/Yelp/bravado){: target="_blank" }
-which generates a dynamic client at runtime, from an OpenAPI definition served by a lakeFS server.
+For Python, this example uses [OpenAPI Generator](hhttps://openapi-generator.tech){: target="_blank" }
+which can generates Python client code for the OpenAPI definition served by a lakeFS server.
 
 ## Table of contents
 {: .no_toc .text-delta }
 
 1. TOC
 {:toc}
 
-## Installing Python Dependencies
+## Install lakeFS Python Client API
+
+Install the Python client using pip:
 
-A complete installation guide is available in the [bravado GitHub repository](https://github.com/Yelp/bravado){: target="_blank" }.
-For our example, we'll simply install it with pip:
 
 ```shell
-pip install bravado==10.6.2
+pip install 'lakefs==<lakeFS version>'
 ```
 
-At the time of writing this guide, the current stable bravado release is `10.6.2`.
-
-
-## Locating the OpenAPI definition for our installation
+The package is available from version 0.34.0.
```

Comment:
Where did you come up with this version?

Reply:
assumed it will be the next release - but I'll have to update it after the release